src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
+/* eslint-disable no-console */
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static deadAnimal(animalName) {
+    Animal.alive = Animal.alive.filter(el => el.name !== animalName);
+  }
+  health = 100;
+
+  constructor() {
```

Comment:
```suggestion
  constructor(name) {
```
Як сказано в умові задачі:
> All instances of the Animal class must have health and a name

Reply:
Оке, поправлю. Спасибо. Не до конца понимаю разницу
к примеру я записал health в конструктор и вне его
Разница получается в чем ? этих двух подход 
Ведь что тот и тот, при создании экземпляра класса - попадет в объект 
С name понятно вроде как, нам нужно создавать  конструктор, чтобы он принимал в себе параметр name 
и уже в конструкторе прописывали this.name = name;

Reply:
name потрібно передавати, щоб при виклику без аргумента у об'єкта поле name не було undefined.

Reply:
Ну раніше в тебе name ніяк не створювалось взагалі

Reply:
и все же вопрос остался для меня открытый
В чем разница этой записи:
```
class Animal {
  health = 100;

  constructor(name) {
    this.name = name;
  }
}
```

от этой

```
class Animal {
  constructor(name) {
    this.health = 100;
    this.name = name;
  }
}
```

В видео, Миша показывал что такая запись допустима
взяв задачи с тестов, что на сайте 
Линтер выдает ошибку 'Parsing error: Unexpected token =', но тесты все проходят